third-party/linshare/src/test/resources/docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
         command: mongod --smallfiles
 
     smtp:
+        container_name: linshare_smtp
```

Comment:
I have to keep it, cause It makes linshare be able to reach to smtp server by the host name. otherwise, I'm fucked up